src/wui/mapview.cc

```diff
@@ -669,12 +669,19 @@ void MapView::increase_zoom() {
 	            Vector2f(get_w() / 2.f, get_h() / 2.f),
 	            animate_map_panning_ ? Transition::Smooth : Transition::Jump);
 }
+void MapView::zoom_to_max() {
+	zoom_around(animation_target_view().view.zoom - kMaxZoom, Vector2f(get_w() / 2.f, get_h() / 2.f),
```

Comment:
```suggestion
	zoom_around(1.f / kMaxZoom, Vector2f(get_w() / 2.f, get_h() / 2.f),
```

Reply:
Thank you! I will look into it again and test it on my end

Reply:
It works. I have changed it before pushing new commit.